Source/WebCore/Modules/webauthn/PublicKeyCredentialRequestOptions.idl

```diff
@@ -28,7 +28,7 @@
 ] dictionary PublicKeyCredentialRequestOptions {
     required BufferSource challenge;
     unsigned long timeout;
-    USVString rpId;
+    DOMString rpId;
```

Comment:
Does this have any web-visible impact? How do we know this didn't break any existing credential request flows (e.g., is it validated by existing test cases?)

Reply:
I'm confident this doesn't have any web-visible impact because the WebAuthn spec has already has limited PublicKeyCredentialRequestOptions.rpId to a valid domain, which is more restrictive than a USVString (full reasoning in https://github.com/w3c/webauthn/pull/2098.) We also already take rpId as a DOMString in another method.